src/applications/mhv/secure-messaging/components/AttachmentsList.jsx

```diff
@@ -113,45 +25,6 @@ const AttachmentsList = props => {
         )}
       </div>
       {editingEnabled && <HowToAttachFiles />}
-
-      {attachFileSuccess &&
-        attachments.length > 0 && (
-          <VaAlert
-            aria-live="polite"
-            aria-label="file successfully attached"
-            ref={attachFileAlertRef}
-            background-only
-            className="file-attached-success vads-u-margin-top--2"
-            data-testid="file-attached-success-alert"
-            disable-analytics
-            full-width="false"
-            show-icon
-            status="success"
-            onCloseEvent={handleSuccessAlertClose}
-          >
-            <p className="vads-u-margin-bottom--0">File attached</p>
-            <button
-              className="close-success-alert-button vads-u-padding--0p5"
-              id="close-success-alert-button"
-              data-testid="close-success-alert-button"
-              aria-label="Close notification"
-              type="button"
-              onClick={() => {
-                setAttachFileSuccess(false);
-                handleSuccessAlertClose();
-              }}
-            >
-              <i
-                className="fas fa-times-circle vads-u-color--black"
-                style={{ fontSize: '2.4rem' }}
-                alt="Close notification icon"
-                aria-hidden="true"
-                role="presentation"
-              />
-            </button>
-          </VaAlert>
-        )}
-
```

Comment:
I'm trying to understand the reasoning behind this change. I'm wondering if it was intentional to distribute one piece of code across four instances in two separate components. I'm open to different approaches if it is absolutely necessary. let me know if I am missing something here